### DISCUSSION TITLE
Header seize on unit - C005F400 - does not match patch files seize

### DISCUSSION THREAD

**Discussion by fthyes** [Mr-MIBonk/M.I.B._More-Incredible-Bash#578] [Q&A] [answered]
Stuck while trying to patch my MIB of my Audi SQ7 running train version MHI2Q_ER_AUG22_P5092. To avoid any troubles with the Bose amplifier I would prefer staying on the FW version which is currently running. I got the latest MIB version 3.6.0 and patched the loader successfully. Now the green menu can be opened and I see the MIB option. So far s good, I tried to install the patches but this does not work unfortunately. 

Here is the relevant part from the logfile.

<log>
MMX-flash---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1ae05bf (15992256)|93...216 (123):LOCKED
0x00ba0000 - C005F400

Found JTB-07419.05.17119308AE Backup, will check if all complete...
Standard-Backup will require ~5 minutes to complete!
Partition info... -> Backup is already there!
SWDL............. -> Backup is already there!
Version info..... -> Backup is already there!
GEM info........ -> Backup is already there!
Folder lists..... -> Backup is already there!
FEC folders...... -> Backup is already there!
HMI folders...... -> Backup is already there!
Shadow files..... -> Backup is already there!
lsd.jxe.......... -> Backup is already there!
Dataset's........ -> Backup is already there!
EEPROM........... -> Backup is already there!
storage1......... -> Backup is already there!
storage2......... -> Backup is already there!
RCC Flashlock.... -> Backup is already there!
RCC.............. -> Backup is already there!
IFS-stage2....... -> Backup is already there!
OFFSET's......... -> Backup is already there!
backing up backup -> Backup part 1 is already there! -> Backup part 2 is already there!

ls -als /net/mmx/mnt/boardbook --> total 192
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:02 .
      64 drwxrwxr-x  5 root      root          32768 Jan 01 00:00 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:18 MU1329
Free space on /net/mmx/mnt/boardbook 820M

OK: Patch folder /net/mmx/fs/sda0/patches/MHI2Q_ER_AUG22_P5092_MU1329_PATCH found on SD card

File  found in patch folder
File MU1329-ifs-root-part2-0x00ba0000-C005F400.ifs nedded to patch this unit
Header seize on unit - C005F400 - does not match patch files seize -  - or stock -  - image
</snip>  

Any advice how to proceed would be really appreciated. 

Cheers
Frank

**Comment by Honeybadger-22** [comment]
Did you get this fixed? It looked like a typo...  I have the exact same issue rn

**Reply by fthyes** [reply, marked as answer]
I asked for help in the telegram channel and received a response a couple of days ago. They also provided a patch but I had no time to test is yet. I will let you know when I know if its working as expected.

UPDATE: Finally I had time to try the patch, android car is working, the unlimited license for the navigation is there as well. Upgrading the NAV DB to 2024 was also not a big deal. I was using the Pre MIB  [Pre-V3.7.1 - MHIG Edition](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/releases/tag/V3.7.1) Patch is attached.

[MC-10176758-0001.pdf](https://github.com/user-attachments/files/17791109/MC-10176758-0001.pdf)
[MHI2Q_ER_AUG22_P5092_MU1329_PATCH.zip](https://github.com/user-attachments/files/17791110/MHI2Q_ER_AUG22_P5092_MU1329_PATCH.zip)

**Comment by raimondas-27** [comment]
Having the same issue, will try your patch
